specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json

```diff
@@ -112,6 +112,160 @@
           }
         }
       }
+    },
+    "/timeseries/": {
+      "get": {
+        "summary": "List time series of each unique user.",
+        "description": "List time series of each unique user.",
+        "operationId": "ListTimeSeries",
+        "parameters": [
+        ],
+        "consumes": [
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation.",
+            "schema": {
+              "type": "array",
```

Comment:
ref to TimeSeriesInfos.
and complete definition of TimeSeriesInfos accordingly.

Reply:
You can refer to TimeSeriesGroups